src/Microsoft.IdentityModel.Tokens/Validation/Validators.Algorithm.cs

```diff
@@ -42,5 +43,65 @@ public static void ValidateAlgorithm(string algorithm, SecurityKey securityKey,
                 });
             }
         }
+
+        /// <summary>
+        /// Validates if a given algorithm for a <see cref="SecurityKey"/> is valid.
```

Comment:
```suggestion
        /// Validates a given algorithm for a <see cref="SecurityKey"/>.
```
"validates if it is valid" is redundant